configuration/amd64/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - mysql-datavolume:/var/lib/mysql
 
   php:
-    image: usdotfhwaops/php:7.6.0
+    image: usdotfhwaops/php:${V2XHUB_VERSION:-develop}
```

Comment:
This looks like a syntax error. Do you mean just ${V2XHUB_VERSION}?
ditto for all occurrences in the PR.
And does this variable need to be in a .env file?

Reply:
No this is not a syntax error. This is how you specify docker-compose environment variables with a default value https://docs.docker.com/compose/environment-variables/env-file/#interpolation. This way we do not have to provide an .env file but if one is used it will overwrite the default values.